src/nih_wayfinding/app/scripts/mapping/map-route-service.js

```diff
@@ -1,97 +0,0 @@
-/**
- *
- */
-(function() {
-    'use strict';
-
-    /* ngInject */
-    function MapRoute($q,
```

Comment:
Removed this abstraction since I needed the setGeojson method in OverviewController in order to draw the route from the cache, and I'm not sure this abstraction was gaining us much.